INSTRUMENTATION_STATUS: class=it.feio.android.omninotes.helpers.BackupHelperTest
INSTRUMENTATION_STATUS: current=1
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: numtests=1
INSTRUMENTATION_STATUS: stream=
it.feio.android.omninotes.helpers.BackupHelperTest:
INSTRUMENTATION_STATUS: test=getBackupFolderPath
INSTRUMENTATION_STATUS_CODE: 1
INSTRUMENTATION_STATUS: class=it.feio.android.omninotes.helpers.BackupHelperTest
INSTRUMENTATION_STATUS: current=1
INSTRUMENTATION_STATUS: id=AndroidJUnitRunner
INSTRUMENTATION_STATUS: numtests=1
INSTRUMENTATION_STATUS: stack=java.lang.ArrayIndexOutOfBoundsException: length=3; index=4
	at it.feio.android.omninotes.helpers.BackupHelper.getBackupFolderPath(BackupHelper.java:338)
	at it.feio.android.omninotes.helpers.BackupHelperTest.getBackupFolderPath(BackupHelperTest.java:206)

INSTRUMENTATION_STATUS: stream=
Error in getBackupFolderPath(it.feio.android.omninotes.helpers.BackupHelperTest):
java.lang.ArrayIndexOutOfBoundsException: length=3; index=4
	at it.feio.android.omninotes.helpers.BackupHelper.getBackupFolderPath(BackupHelper.java:338)
	at it.feio.android.omninotes.helpers.BackupHelperTest.getBackupFolderPath(BackupHelperTest.java:206)

INSTRUMENTATION_STATUS: test=getBackupFolderPath
INSTRUMENTATION_STATUS_CODE: -2
INSTRUMENTATION_RESULT: stream=

Time: 0.113
There was 1 failure:
1) getBackupFolderPath(it.feio.android.omninotes.helpers.BackupHelperTest)
java.lang.ArrayIndexOutOfBoundsException: length=3; index=4
	at it.feio.android.omninotes.helpers.BackupHelper.getBackupFolderPath(BackupHelper.java:338)
	at it.feio.android.omninotes.helpers.BackupHelperTest.getBackupFolderPath(BackupHelperTest.java:206)

FAILURES!!!
Tests run: 1,  Failures: 1


INSTRUMENTATION_CODE: -1
